File: javascript/set06/problems/problem1.js
// Print distance between the first 100 prime numbers
function isPrime(n){
    if (n < 2)
        return false;  
    if (n == 2)
        return true;
    let half = Math.sqrt(n);
    
    for(let i = 2; i <= half; i++){
        if (n % i === 0)
        {
            return false;
        }
    }
    return true;
}
let array_prime = []
let n = 0;
    let i = 2;
    
    while(n < 100)
    {
        if (isPrime(i))
        {
            array_prime.push(i);
            n++;
        }
        i++;
    }
console.log(array_prime[99]-array_prime[0])

File: python/set06/problems/problem1.py
# Print the distance between the first hundred prime numbers
def is_prime(n):
    if n < 2:
        return False
    for i in range(2, int(n**0.5) + 1):
        if n % i == 0:
            return False
    return True
primes = []
num = 2
while len(primes) < 100:
    if is_prime(num):
        primes.append(num)
    num += 1
print(primes[99]-primes[0])
